File: dislinktxws-front/src/app/model/newDislikeDTO.ts
import { Post } from "./post.model";

interface NewDislikeDTOInterface{
    idProfileWhoDislike : number;
    dislikedPost : Post;
    
}
export class NewDislikeDTO implements NewDislikeDTOInterface{
    idProfileWhoDislike : number;
    dislikedPost : Post;
    constructor(obj:NewDislikeDTOInterface){
        this.idProfileWhoDislike=obj.idProfileWhoDislike;
        this.dislikedPost=obj.dislikedPost;
    } 
}

File: DislinktXWS/src/main/java/com/example/DislinktXWS/DTO/NewDislikeDTO.java
package com.example.DislinktXWS.DTO;

import com.example.DislinktXWS.model.Post;

public class NewDislikeDTO {
    private Long idProfileWhoDislike;
    private Post dislikedPost;

    public Long getIdProfileWhoDislike() {
        return idProfileWhoDislike;
    }

    public void setIdProfileWhoDislike(Long idProfileWhoDislike) {
        this.idProfileWhoDislike = idProfileWhoDislike;
    }

    public Post getDislikedPost() {
        return dislikedPost;
    }

    public void setDislikedPost(Post dislikedPost) {
        this.dislikedPost = dislikedPost;
    }
}
